content/english/facts/soy-contains-estrogen.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Soy Contains Estrogen"
+date: 2019-09-28T19:08:41+01:00
+lastmod: 2019-09-28T19:08:41+01:00
+draft: false
+description: "Is it true that: \"Soy contains estrogen\"?"
+weight: 100
+---
+
+# Popular Fact
+
+> Soy contains estrogen and is akin to hormone replacement therapy
+
+or
+ 
+> Soy contains *phytoestrogens* which are completely different and have no effect on humans
+
+# Reality 
+
+> Soy contains phytoestrogens which may interact with the human body in a variety of ways (including hormonally).   
+> - TalkVeganToMe
+
+See below for a more detailed explanation.  
+See our article on [Soy is bad for you?]({{<ref "/facts/soy-is-bad-for-you">}}) for more on what this means.  
+
+## What are phytoestrogens and isoflavones?
+> Phytoestrogens are naturally-occurring plant compounds that are structurally and/or functionally similar to mammalian estrogens and their active metabolites.
+> - [The pros and cons of phytoestrogens - Front Neuroendocrinol - 2010 - p2](https://web.archive.org/web/20190928181945/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3074428/pdf/nihms271669.pdf)
+
+### What Isoflavones does soy contain?
+> Soy contains the isoflavones:  
+> - Biochanin A   
+> - Clycitein   
+> - Daidzein  
+> - Formononetin  
+> - Genistein    
+> - [The pros and cons of phytoestrogens - Front Neuroendocrinol - 2010 - p2](https://web.archive.org/web/20190928181945/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3074428/pdf/nihms271669.pdf)
+
+## How do isoflavones interact with the body?
+> Some isoflavones, most notably genistein, inhibit pathways important for cell growth and proliferation, an effect which affects multiple organ systems.  
+> - [The pros and cons of phytoestrogens - Front Neuroendocrinol - 2010 - p4](https://web.archive.org/web/20190928181945/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3074428/pdf/nihms271669.pdf)
+
+In other words, it's possible for human cell growth to be increased by phytoestrogens.
+
+> Phytoestrogens are plant‐derived dietary compounds with structural similarity to 17‐β‐oestradiol (E2), the primary female sex hormone. This structural similarity to E2 enables phytoestrogens to cause (anti)oestrogenic effects by binding to the oestrogen receptors.  
+> [...]  
```

Comment:
Makes it sound like Phytoestrogens are *ONLY* plant derived.

Reply:
Are they not? https://en.wikipedia.org/wiki/Phytoestrogen

Reply:
https://pubs.acs.org/doi/full/10.1021/jf801344x

Reply:
No way! I did not know that. Makes a lot of sense though. Thanks. I'll add!